xarray/tests/test_cftimeindex.py

```diff
@@ -830,7 +830,6 @@ def test_cftimeindex_add_timedeltaindex(calendar) -> None:
 @pytest.mark.parametrize(
     "freq,units",
     [
-        ("D", "D"),
```

Comment:
Now that `Day` is non-`Tick`-like, multiplying by a `float` is no longer valid in pandas, so shift with a float value and `Day` offset will raise an error:
```
>>> import pandas as pd
>>> times = pd.date_range("2000", periods=5)
>>> times.shift(0.5, "D")
Traceback (most recent call last):
  File "<python-input-2>", line 1, in <module>
    times.shift(0.5, "D")
    ~~~~~~~~~~~^^^^^^^^^^
  File "/Users/spencer/software/pandas/pandas/core/indexes/datetimelike.py", line 514, in shift
    start = self[0] + periods * self.freq
                      ~~~~~~~~^~~~~~~~~~~
TypeError: unsupported operand type(s) for *: 'float' and 'pandas._libs.tslibs.offsets.Day'
```
A similar error is raised now for attempting to shift a `CFTimeIndex` in this way.